src/templates/nl/accessibility_statement.html

```diff
@@ -0,0 +1,120 @@
+  </p>
+
+  <p>
+    Op dit moment hebben we het testen van ondersteunende technologiehardware of -software niet voltooid, maar hebben we hulpmiddelen gebruikt, zoals hierboven vermeld, om te proberen eventuele problemen met deze apparaten op te lossen.
+  </p>
+</section>
+
+<section>
+  <h2 id="technical-specifications"><a href="#technical-specifications" class="anchor-link">Technische Specificaties</a></h2>
+
+  <p>
+    De website is gebouwd op goed gestructureerde, semantische HTML (inclusief <a href="https://www.w3.org/WAI/standards-guidelines/aria/">WAI-ARIA</a> waar van toepassing), aangevuld met CSS voor styling en beperkt JavaScript voor sommige interactieve componenten. De website maakt gebruik van moderne ontwikkelingspraktijken zoals CSS Grid, Fetch en WOFF2 Lettertypen, die niet worden ondersteund in oudere browsers zoals IE 11. De website is nog steeds volledig functioneel in die browsers, maar is misschien niet zo visueel aantrekkelijk. Er zijn geen extra plug-ins of bibliotheken nodig om de website te bekijken.
+  </p>
+  <p>
+    De Web Almanac PDF volgt dezelfde toegankelijkheidsnormen als de website en is bovendien structureel getagd om de toegankelijkheid te bevorderen.
+  </p>
+</section>
+
+<section>
+  <h2 id="assessment-approach"><a href="#assessment-approach" class="anchor-link">Beoordelingsaanpak</a></h2>
+
+  <p>
+    Zelfevaluatie: de inhoud werd geëvalueerd door het Web Almanac-team met behulp van hun eigen expertise en een aantal hulpmiddelen, waaronder de <a href="https://wave.webaim.org/">WAVE Toegankelijkheidstool</a>, de <a href="https://hacks.mozilla.org/2019/10/auditing-for-accessibility-problems-with-firefox-developer-tools/">Firefox Toegankelijkheidsinspector</a> en de <a href="https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd" rel="nofollow"> AXE Chrome-plug-in</a>. De Web Almanac PDF is getest met een toegankelijkheidsscan in Acrobat Reader Pro DC.
+  </p>
+</section>
+
+<section>
+  <h2 id="feedback"><a href="#feedback" class="anchor-link">Feedback</a></h2>
+
+  <p>
+    We verwelkomen feedback over de toegankelijkheid van deze website. De beste manier om ons te bereiken is door <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues">een probleem op GitHub te melden</a> of ons een e-mail te sturen op <a href="mailto:team@httparchive.org">team@httparchive.org</a>. We verwelkomen ook <span lang="en">pull requests</span> voor degenen die rechtstreeks willen bijdragen. De Web Almanac is een gemeenschapsproject, beheerd door vrijwilligers, en daarom kunnen we geen tijdsbestek garanderen voor het reageren op feedback, maar zullen we zo snel mogelijk reageren.
+  </p>
+</section>
+{% endblock %}
```

Comment:
Nit, files should end in newlines

```suggestion
{% endblock %}

```